Gems/Atom/Feature/Common/Code/Source/AuxGeom/AuxGeomFeatureProcessor.cpp

```diff
@@ -157,15 +157,9 @@ namespace AZ
             m_fixedShapeProcessor->SetUpdatePipelineStates();
         }
 
-        void AuxGeomFeatureProcessor::OnRenderPipelineAdded(RPI::RenderPipelinePtr pipeline)
+        void AuxGeomFeatureProcessor::OnRenderPipelineChanged(RPI::RenderPipeline*, RPI::SceneNotification::RenderPipelineChangeType)
```

Comment:
Yes, I changed that intentionally. The function OnSceneRenderPipelinesChanged() should also be called when pass change event is triggered since it may change the output layout which need pipeline states update.